data/modules/FindPerson/FindPerson.lua

```diff
@@ -127,8 +127,8 @@ local onChat = function (form, ref, option)
 			client      = ad.client,
 			wanted      = ad.wanted,
 			company     = ad.company,
-			location    = ad.location:GetStarSystem().path,
-			destination = ad.location,
+			location    = ad.location,
+			destination = ad.location:GetStarSystem().path,
```

Comment:
Long after-the-fact correction: `SystemPath:GetStarSystem()` followed by `.path` is an incorrect form to use here. It potentially requires generating the entire StarSystem object again to fill the cache, which has performance implications.

Instead, code should always use `SystemPath:SystemOnly()` to retrieve the path to the containing system from an existing body path.

Reply:
I do wonder if there are places in the master branch that could do with this optimization? I can't recall myself being aware of this distinction...